instance/models/instance.py

```diff
@@ -89,6 +92,18 @@ def save(self, *args, **kwargs):
             'instance_id': self.pk,
         })
 
+    @classmethod
+    def can_manage(cls, user):
+        """
+        Returns true if the user is allowed to manage all instances.
+
+        User must:
+        * be a staff member, or
+        * have the "manage_all" permission
+        """
+        permission = '{}.{}'.format(cls._meta.app_label, "manage_all")
+        return user.is_staff or user.has_perm(permission)
```

Comment:
@pomegranited I agree with @bradenmacdonald that it's better to just check for the permission. The `is_staff` is normally really just to control who can login in the Django admin as far as I know. We could just give superuser privileges to everyone in the team for now. 

Another alternative if we don't want to use the superuser privilege is to create an "admin" group in the django admin, and grant the "manage all" permission to that group, then add the whole team to that group.